lib/urlwatch/storage.py

```diff
@@ -507,3 +517,72 @@ def clean(self, guid):
             return result
 
         return 0
+
+
+class CacheRedisStorage(CacheStorage):
+    def __init__(self, filename):
+        super().__init__(filename)
+
+        if redis is None or msgpack is None:
+            raise ImportError('redis + msgpack are missing')
+
+        self.db = redis.from_url(filename)
+
+    def _make_key(self, guid):
+        return 'guid:' + guid
```

Comment:
@scottmac why do you prepend `'guid: '` to the key? Is it purely to make it look nicer, or is there a technical reason?

I'm curious, as I'll probably need to make changes to all cache backends, and I'm not very familiar with redis.